thoth/slo_reporter/sli_python_knowledge_graph/sli_pypi_knowledge_graph.py

```diff
@@ -23,12 +23,14 @@
 
 import numpy as np
 
+import pandas as pd
```

Comment:
do we still need those imports?

Reply:
Good catch, thanks :100:

Reply:
https://github.com/thoth-station/slo-reporter/pull/130/commits/725345d6f54ea23090d7c1bab0b4a7e1a0e837b9